permissions/Base.permission

```diff
@@ -185,3 +185,7 @@ mkdir     ${HOME}/Music
 blacklist ${HOME}/Music
 mkdir     ${HOME}/Videos
 blacklist ${HOME}/Videos
+mkdir     ${HOME}/Public
+blacklist ${HOME}/Public
+mkdir     ${HOME}/Templates
+blacklist ${HOME}/Templates
```

Comment:
Worth create and blacklist also Desktop. @Tomin1 ?

Reply:
Makes sense unless we explicitly disallow every dir access by default? I've not been involved in this long enough to understand this completely yet..

Reply:
Note: We actually have a mention of Desktop in WebView.permission but I'm not sure it should be there and I think we should get rid of that. It's probably not used in practice but again one of those, it happens to access it but doesn't use it so it's there. If we blacklist it here and we keep it there, then there needs to be noblacklist rule.

That said, maybe we should blacklist it to discourage use. By default creating files to wrong places causes them disappear when the sandbox is destructed (application is closed) but this way you can't create files to some of the wrong places which gives developer (or user) immediate feedback instead of wondering what happened.

Reply:
@Tomin1 or we change the behavior of `XDG_DESKTOP_DIR` to point to where the user app could create additional shortcuts? See https://github.com/sailfishos/sailjail-permissions/pull/62#discussion_r613094062

Reply:
> Note: We actually have a mention of Desktop in WebView.permission but I'm not sure it should be there and I think we should get rid of that. It's probably not used in practice but again one of those, it happens to access it but doesn't use it so it's there. If we blacklist it here and we keep it there, then there needs to be noblacklist rule.

Having "Desktop" in the WebView.permission sounds wrong indeed.

Reply:
Unless we want to give access via that to the `$HOME/.local/share/applications` to add URL desktop entries there to be visible on _our_ desktop?

Reply:
Eventually resulting in modifying `.config/user-dirs.dirs` as well.

Reply:
Correct and that's another task and not related to this per se.

Reply:
Not related I agree but a worthy issue to raise here as well but not to venture into.

Reply:
NB That "Desktop in WebView.permission" is there due to: initial rules were generated semi-automated and in a mass -> it eliminated an access failure from logs. If it is known to be wrong / lack of it would be inconsequential, it should be removed.

Reply:
> Unless we want to give access via that to the `$HOME/.local/share/applications` to add URL desktop entries there to be visible on _our_ desktop?

I don't think that would make sense since these files should be in the paths which the xdg-menu spec specifies:
https://specifications.freedesktop.org/menu-spec/menu-spec-latest.html#paths

Also the intend behind this is different IMHO. It is more a place to put desktop files there that are already in the menu or link common folder etc.

If we want to support adding new desktop entries that would be another task see:
https://specifications.freedesktop.org/menu-spec/menu-spec-latest.html#menu-editing